server/documents/modules/calendar.html.eco

```diff
@@ -476,18 +476,18 @@ themes      : ['Default']
       <div class="code" data-type="javascript">
       $('#disableddates_calendar')
         .calendar({
-            initialDate: new Date(2018,11,1),
+            initialDate: new Date('2018-11-1'),
             disabledDates: [{
-                    date: new Date(2018,11,22),
+                    date: new Date('2018-11-22'),
                     message: 'xmas gift shopping'
                 },
                 {
-                    date: new Date(2018,11,25),
+                    date: new Date('2018-11-25'),
                     message: 'Santa Clause is coming to town',
                     inverted: true,
                     variation: 'basic'
                 },
-                new Date(2018,11,31)  /* No Reason. Everybody knows why ;) */
+                new Date('2018-11-31')  /* No Reason. Everybody knows why ;) */
```

Comment:
This is wrong. November has 30 days, a Date('2018-11-31') results in a wrong date and the intention is to show December 31st, so Date(2018,11,31) is correct

Reply:
You're right. I forgot that the natural JavaScript date arguments and haven't check the JavaScript date documentation. 

In my opinion, the FUI example looks confusing with that usage, because many people are used to with `Date(YYYY-mm-dd)` instead of `Date(yyyy,mm,dd)`, and can be interpreted incorrectly like me.

Reply:
I think passing string date is reasonable, because 0-origin month is very confusing.

Reply:
Yes. It also gets misinterpreted while reading fast and having a quick look for references.

Reply:
Anyway: '2018-11-31' is still wrong, becasue it does not exist 😉 
And: Santa Clause still comes in December, not November 😉 So, if you want to have the string approach seen in the docs, then at least make sure the dates are the same as before (means ??,11,?? has to be ??-12-?? then). You currently just replaced the commas by dashes which always results in a different month as before.

Reply:
I'll update the PR if you guys agree to change the date format. Otherwise, I'll end up with two commits which are not going to be merged :smiley:

Reply:
If Christmas is still happening in December, fine your me 🤣